firebase-firestore/src/androidTest/java/com/google/firebase/firestore/TransactionTest.java

```diff
@@ -513,38 +499,32 @@ public void testUpdatePOJOTransactionally() {
   }
 
   @Test
-  public void testHandleReadingOneDocAndWritingAnother() {
+  public void retriesWhenDocumentThatWasReadWithoutBeingWrittenChanges() {
```

Comment:
Though not required, conventionally we've written these methods with a "test" prefix.